src/server/lib/format.js

```diff
@@ -13,32 +16,39 @@ const EXPENDITURE_CATEGORIES = {
   ec7: '7-Administrative and Other'
 }
 
-function currency (value, optional = false) {
-  if (optional && value == null) {
-    return value
-  }
+/**
+ * Normalize casing of single word values.
+ * This is especially useful for "Yes"/"No" responses.
+ *
+ * @param {string} value
+ * @returns {string}
+ */
+function capitalize (value) {
```

Comment:
nit: Would it make sense to name this `capitalizeFirst` or something similar to disambiguate it from something like `toUpperCase()`?

Reply:
Oof, you're right.  This name is ambiguous.  I'll update!